less/index.less

```diff
@@ -145,11 +145,24 @@ body {
   height: 100%;
 }
 
-//Variables
+// Start Variables
 @font-size: 6rem;
 @primary-color: #428bca;
+// End Variables
+
+// Start of Mixins
+.my-hover(){
+&:hover {
+      background-color: blue;
+}
+}
+
+.mb {
```

Comment:
Don't forget to put parentheses when creating a mixin because when your LESS file is compiled to CSS, it will include the mixins you created and they'll be treated as classes.